lib/components/map/enhanced-stop-marker.js

```diff
@@ -120,60 +120,54 @@ class EnhancedStopMarker extends Component {
       <Icon style={{ height: 32, width: 32 }} type="map-marker" />
     )
 
-    const icon = divIcon({
-      className: '',
-      html: ReactDOMServer.renderToStaticMarkup(
-        <BaseStopIcon
-          active={id === activeStopId}
-          mainColor={color}
-          secondaryColor={getComplementaryColor(tinycolor(color))}
-          // Show actual stop name on hover for easy identification.
-          thick={activeStopId === id}
-          title={getStopName(stop)}
-        >
-          {stopIcon}
-        </BaseStopIcon>
-      ),
-      // Anchor of [0, 0] places the top-left corner of the bubble at the stop location.
-      // Instead, we want the tip of the caret at the bottom center of the bubble
-      // to be at the stop location.
-      // Add some margins so the stop marker (which may be unintentionally offset) remains visible.
-      iconAnchor: [
-        bubblePixels(this.props) / 2 + 4,
-        bubblePixels(this.props) + caretVisibleHeight + 8
-      ]
-    })
+    const icon = (
+      <BaseStopIcon
+        active={id === activeStopId}
+        mainColor={color}
+        secondaryColor={getComplementaryColor(tinycolor(color))}
```

Comment:
Remove unused prop.

Reply:
Removed in d3575aa5c7d8ea8b45d8959c9d74f64ac03b0274